src/index.js

```diff
@@ -86,6 +86,10 @@ class Keyv extends EventEmitter {
 			.then(() => true);
 	}
 
+	async has(key) {
+		return Boolean(await this.get(key, { raw: true }));
```

Comment:
Here I would probably check like this

```js
(typeof await this.get(key, { raw: true }) === 'object')
```

Although not sure that's necessarily better. Any specific reason you chose to wrap with `Boolean()`?

// CC @sindresorhus any input would be appreciated here 🙏

Reply:
Well it's just something I do so I would know I want to know if that value exists or not. If I use `typeof` it look to me like I'm checking for `object` specifically but what I actually want anything except `null`/`undefined`. I'd also like Sindres input on this.

Reply:
> If I use typeof it look to me like I'm checking for object specifically but what I actually want anything except null/undefined.

That's exactly why I don't think it should be written like this. I think you **should** be checking for an object explicitly.

Using `{raw: true}` will return the raw keyv object if it's set or `undefined` if it's not. So you literally want to say "Does this function return an object." or "Does this function not return `undefined`"

So 

```js
typeof (await this.get(key, { raw: true })) === 'object'
```

or 

```js
(await this.get(key, { raw: true })) !== undefined
```

are semantically correct.

What you're currently doing:

```js
Boolean(await this.get(key, { raw: true }));
```

is just saying "Is this value truthy".

Which happens to work in this scenario, an object will be truthy and `undefined` will be falsey. However it's not checking the exact value we know it should be. It's less explicit and it's possible if we change the way data is structured internally it could break.

Does that make sense?